infra/cifuzz/fuzz_target.py

```diff
@@ -121,6 +124,32 @@ def is_reproducible(self, test_case):
         return True
     return False
 
+  def is_crash_valid(self, test_case):
+    """Checks if a crash was introduced by the pull request.
+
+    NOTE: If no old_build_target is specified the crash is assumed introduced
+    by the pull request.
+
+    Args:
+      test_case: The path to the test_case that triggered the crash.
+
+    Returns:
+      True if the crash was introduced by the current pull request.
+    """
+    if not self.old_build_path:
+      return True
+    exists_in_pr = self.is_reproducible(test_case,
```

Comment:
so if there is no old build specify, we don't even bother to verify reproducibility on the current build? seems wrong :)

If we already checked the reproducibility in some other place, this call isn't needed I guess and function name should be something like `is_regression`

Reply:
Good catch!!

Reply:
Does my patch  match up to what you expected?